crates/cargo-test-macro/Cargo.toml

```diff
@@ -1,13 +1,20 @@
 [package]
 name = "cargo-test-macro"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Why bumping to `0.1.1`?

Reply:
This was to make `validate-version-bump.sh` happy.

Reply:
Thanks. Got it. Let's bump it to 0.2.0 so we don't need to bump it again.

BTW, I just published them to crates.io and am now waiting for infra team to accept ownership transfer.

* https://crates.io/crates/cargo-test-support
* https://crates.io/crates/cargo-test-macro